app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,62 @@
+class PasswordResetsController < ApplicationController
+  before_action :find_user, :valid_user, :check_expiration,
+    only: [:edit, :update]
+
+  def new; end
+
+  def create
+    @user = User.find_by email: params[:password_reset][:email].downcase
+    if @user
+      @user.create_reset_digest
+      @user.send_password_reset_email
+      flash[:info] = t "pass_reset"
+      redirect_to root_url
+    else
+      flash.now[:danger] = t "e_not_found"
```

Comment:
đừng viết tắt chị, viết rõ ràng cái gì not_found đi ạ, nếu mà là khai báo not found chung thì chỉ viết là not_found thôi